src/components/ha-lawn_mower-state.ts

```diff
@@ -0,0 +1,91 @@
+import "@material/mwc-button";
+import { CSSResultGroup, LitElement, css, html } from "lit";
+import { customElement, property } from "lit/decorators";
+import { supportsFeature } from "../common/entity/supports-feature";
+import {
+  LawnMowerEntity,
+  LawnMowerEntityFeature,
+  LawnMowerEntityState,
+} from "../data/lawn_mower";
+import { HomeAssistant } from "../types";
+
+type LawnMowerAction = {
+  action: string;
+  service: string;
+  feature: LawnMowerEntityFeature;
+};
+
+const LAWN_MOWER_ACTIONS: Partial<
+  Record<LawnMowerEntityState, LawnMowerAction>
+> = {
+  mowing: {
+    action: "dock",
+    service: "dock",
+    feature: LawnMowerEntityFeature.DOCK,
+  },
+  docked: {
+    action: "start_mowing",
+    service: "start_mowing",
+    feature: LawnMowerEntityFeature.START_MOWING,
+  },
+  paused: {
+    action: "resume_mowing",
+    service: "start_mowing",
+    feature: LawnMowerEntityFeature.START_MOWING,
+  },
+};
+
+@customElement("ha-lawn_mower-state")
```

Comment:
Since this doesn't render the state, but a button with a different action should we rename it?

Reply:
`lawn_mover-action` or something?